app/assets/javascripts/todos.js.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
We can axe this file, seeing as you're writing an API-only project that has no views or other assets.